src/components/PoweredBy/PoweredBy.tsx

```diff
@@ -12,7 +12,7 @@ type Props = {
   cssClasses: CSSClasses;
 };
 
-const PoweredBy = ({ url, theme, cssClasses }: Props) => (
+const PoweredBy = ({ url, theme, cssClasses }: Props): React.ReactNode => (
```

Comment:
```suggestion
const PoweredBy React.FC<Props> = ({ url, theme, cssClasses })  => (
```

this style seems less repetitive to me, WDYT?

Reply:
This doesn't work with our Preact setup.

Reply:
No opinion on this but one issue is that `eslint-plugin-react`'s `prop-types` rule does not support this pattern and will mark `url`, `theme` and `cssClasses` as missing in proptypes. 

```js
const PoweredBy: React.FC<Props> = ({ url, theme, cssClasses }: Props) => (
```
would works but it makes us repeat `Props` :neutral_face:

Reply:
(I should definitively add the support for this pattern in the plugin)